github/WorkflowJob.py

```diff
@@ -0,0 +1,201 @@
+############################ Copyrights and license ############################
+#                                                                              #
+# Copyright 2020 Steve Kowalik <steven@wedontsleep.org>                        #
```

Comment:
I did not write this file, please update this accordingly.

Reply:
Sorry about that. I copied the files when I made the new ones to make it easier to match the style and missed this. I have corrected it in 12af8b42f3a7e141d572c1cbe1cecf5ab38a067a.